### ISSUE TITLE
space in fastq header

### ISSUE THREAD

**Issue by krejciadam** [brwnj/fastq-multx#13]
Hi!
  First of all, thank you for this useful tool. I have a little remark: 

When demultiplexing from separate R1, R2, I1, I2 files, the resulting demultiplexed fastq files get their respective barcode sequences added to the headers separated by a space. Depending on the original header format, this sometimes results in a non-standard header like this:

@NB551430:403:HGKFLBGXG:1:11101:16941:1077/1 GTAGAGGA

This causes problems in subsequent processing with some tools. I would wish to have the option to keep the headers untouched and not to have the barcodes added at all (Now I have to remove them in a subsequent step). 

Thank you!

**Comment by brwnj** [comment]
I agree that adding this option could be done, but disagree that space delimited metadata in the header is somehow non-standard. It's quite standard and any tool that can't deal with it is the issue.

I'll try to look into this as an option if I have some free time.
